index.js

```diff
@@ -16,3 +16,59 @@ async function start() {
   // Now try and complete the program.
   process.exit();
 }
+fconst readline = require("readline");
```

Comment:
You have your readline set-up twice and it's causing your program to error out, also in this instance you have a "f" in front of "const"... so you can delete the stuff from lines 19 to 26